.changeset/eleven-drinks-live.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/functions": major
```

Comment:
```suggestion
"@vercel/functions": minor
```

Reply:
why not a major? this is a breaking change